File: client/src/entities/Option.ts

class Option {
	id: number;
	productId: number;
	name: string;
	price: number;

    constructor(id: number, productId: number, name: string, price: number) {
        this.id = id;
        this.productId = productId;
        this.name = name;
        this.price = price;
    }
}

export { Option }

File: server/internal/entity/data/option.go
package data

import "miadok-chaladok/internal/entity"

type Option struct {
	entity.GormModel
	ProductID uint    `json:"productId"`
	Name      string  `json:"name"`
	Volume    float32 `json:"volume"`
	Price     float32 `json:"price"`
}
